### PR TITLE
Fix commands in Connect with Port Forwarding guide

### PR DESCRIPTION
**Pull request by lukaszo**
fixes:
* added -f flag to command create
* changed not existent flag -t to --template

### CODE REVIEW

File: docs/user-guide/connecting-to-applications-port-forward.md [hunk 1]
```diff
@@ -22,8 +22,9 @@ redis-master   2/2       Running   0          41s
 
 The Redis master is listening on port 6379, to verify this,
 
-```shell{% raw %}
-$ kubectl get pods redis-master -t='{{(index (index .spec.containers 0).ports 0).containerPort}}{{"\n"}}'
+```shell
```
Comment (MHBauer) [inline docs/user-guide/connecting-to-applications-port-forward.md]: This newline isn't needed. Just makes the source box extra big.

Comment (lukaszo) [inline docs/user-guide/connecting-to-applications-port-forward.md]: fixed

**Comment by lukaszo** [comment]
cc @kubernetes/kubernetes-docs-maintainers

**Comment by thockin** [comment]
Thanks!

**Pull request merged**